### ISSUE TITLE
Feature Request: Support LayerNormalization in TFLite

### ISSUE THREAD

**Issue by christian-steinmeyer** [ARM-software/armnn#724]
As integral part of many transformer models, it'd be great if armnn supported [LayerNormalization](https://www.tensorflow.org/api_docs/python/tf/keras/layers/LayerNormalization).

**Comment by TeresaARM** [comment]
Hello @christian-steinmeyer 
Arm NN has a Normalization layer
/armnn/src/armnn/layers/NormalizationLayer.cpp

Which error are you seeing? Maybe we are missing some parameter?

**Comment by christian-steinmeyer** [comment]
Hi! To add a bit of context, I'm working on pruning a tensorflow model and using xnnpack to accelerate inference with it on Android. To that end, I have a model (see below), convert it to tflite using `tf.lite.Optimize.EXPERIMENTAL_SPARSITY` and run the [tensorflow benchmark tool](https://www.tensorflow.org/lite/performance/measurement#android_benchmark_app) on it. 

Here's the toy model I'm using:
```py
model = keras.Sequential(
        [
            keras.layers.InputLayer(input_shape=(28, 28, 1)),
            keras.layers.Conv2D(filters=12, kernel_size=(3, 3), activation='relu'),
            keras.layers.MaxPooling2D(pool_size=(2, 2)),
            keras.layers.Conv2D(1024, (7,7)),
            keras.layers.Flatten(),
            keras.layers.LayerNormalization(),  # works without, but not with this normalization layer
            keras.layers.Dense(10, activation="softmax"),
        ]
    )
    # ...
    converter = tf.lite.TFLiteConverter.from_keras_model(model)
    converter.optimizations = [tf.lite.Optimize.DEFAULT, tf.lite.Optimize.EXPERIMENTAL_SPARSITY]
    converter.target_spec.supported_types = [tf.float16]
    converter.allow_custom_ops = True
    tflite_model = converter.convert()
```

When I include the `LayerNormalization` layer, running the benchmark tool, I get the following log output: `tflite  : Failed to apply XNNPACK delegate.` vs `Replacing 12 out of 12 node(s) with delegate (TfLiteXNNPackDelegate) node, yielding 1 partitions for the whole graph` without this layer.

**Comment by MikeJKelly** [comment]
Hi @christian-steinmeyer 

the XNNPACK delegate is a TFLite delegate, (see https://github.com/tensorflow/tensorflow/tree/master/tensorflow/lite/delegates/xnnpack ) not the ArmNN delegate. From looking at the XNNPACK page it doesn't seem like they support LayerNormalization yet but they may be able to help you there. 

Best regards,
Mike

**Comment by christian-steinmeyer** [comment]
Thanks Mike!
I thought somehow armnn was responsible. Thanks for the pointer. I think this belongs to the tensorflow repo then. Best,
Chris
